src/components/CompactFarm/CompactFarm.Props.ts

```diff
@@ -8,5 +8,8 @@ export interface ICompactFarmProps {
   iconName?: string;
   serverOnClick?: (groupId: any, serverId: any) => void;
   groupOnClick?: (groupId: any) => void;
+  editRoles?: boolean;
```

Comment:
i second this suggestion. better to see that it is a boolean directly from the name